src/components/CurriculumComponents/CurriculumHeader/CurriculumHeader.tsx

```diff
@@ -59,12 +61,18 @@ const CurriculumHeader: FC<CurriculumHeaderPageProps> = ({
   }`;
 
   let pageTitle: string;
+  const keyStageStrings: string[] = [];
+  if (keyStages.includes("ks1")) keyStageStrings.push("KS1");
+  if (keyStages.includes("ks2")) keyStageStrings.push("KS2");
+  if (keyStages.includes("ks3")) keyStageStrings.push("KS3");
+  if (keyStages.includes("ks4")) keyStageStrings.push("KS4");
+  const keyStageString = keyStageStrings.join(" & ");
   switch (phase.slug) {
     case "primary":
```

Comment:
Do we still need the case if the`pageTitle` is the same for both cases?

Reply:
Added ticket to fix in `CUR-923` (just so we can get this merged)